test/lit/passes/signature-refining_gto.wat

```diff
@@ -1,8 +1,10 @@
-;; RUN: foreach %s %t wasm-opt --nominal --signature-refining --gto --roundtrip -all -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt --nominal --signature-refining --gto                       --roundtrip -all -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt           --signature-refining --gto --remove-unused-types --roundtrip -all -S -o - | filecheck %s --check-prefix ISOREC
```

Comment:
Please add a comment as to what this is testing (contrasted to nominal on the line before).